dist/common/systemd/scylla-helper.slice

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Slice used to run companion programs to Scylla. Memory, CPU and IO restricted
+DefaultDependencies=no
+Before=slices.target
+Wants=-.slice
+After=-.slice
+
+[Slice]
+MemoryAccounting=true
+IOAccounting=true
+CPUAccounting=true
+
+CPUWeight=10
+IOWeight=10
+
+# MemoryHigh is the throttle point
+MemoryHigh=4%
+# MemoryMax is the OOM point. As Scylla reserves 7% by default, the other two percent goes to
+# the kernel and other non contained processes
+MemoryMax=5%
+# Systemd deprecated settings BlockIOWeight, MemoryLimit and CPUShares. But they are still the ones used in RHEL7
+# Newer SystemD wants MemoryHigh/MemoryMax, IOWeight and CPUWeight instead. Luckily both newer and older SystemD seem to
+# ignore the unwanted option so safest to get both. Using just the old versions would work too but
+# seems less future proof. Using just the new versions does not work at all for RHEL7/
+MemoryLimit=5%
+CPUShares=10
+BlockIOWeight=10
+
+# Run for at most 25% of the time per each 1ms increment per CPU. That does cap
+# the maximum CPU usage at 25% which is less ideal, but that is a requirement to
+# use CPUQuotaPeriodSec, which we want to make sure that the process does not consume
+# an unreasonable amount of the task quota.
+CPUQuota=25%
+CPUQuotaPeriodSec=1ms
+
```

Comment:
This needs significant testing. These low numbers can increase scheduling overhead. Also, a quota means that idle time cannot be used, and it applies to non-Scylla cores too.

Perhaps using priority is better. We can both increase Scylla's priority and reduce helper priority. Low priority can still take unused bandwidth on both Scylla and non-Scylla cores, and likely has fewer interactions with the scheduler, since it doesn't need to set additional timers.

Reply:
The whole thing needs significant testing, and I was wondering if we can run a preview in QA so we can get more coverage than what I can locally.

I agree with what you say about the disadvantage of the quota, I said as much in the commit message. I ended up deciding to use it just so we can try to guarantee the task quota is not violated too.

However to make this move I'd propose we remove this altogether in favor of the Weight system only, and as we gain more experience with it if we still feel the need to add Quotas we can revisit the decision.